srcCpp/PerftestTransport.cxx

```diff
@@ -770,8 +771,12 @@ std::string PerftestTransport::helpMessageString()
     }
     oss
     << "\t-multicastAddr <address>      - Use multicast to send data and set\n"
-    << "\t                                the input <address> as the multicast\n"
-    << "\t                                address for all the topics.\n"
+    << "\t                                the input <address>|<addr,addr,addr>\n"
+    << "\t                                as the multicast addresses for the "
+    << "\t                                three topics in the application.\n"
+    << "\t                                If only one address is set, the next "
```

Comment:
If only one address is provided, that one and the 2 consecutive ones will be used for the 3 topics used by Perftest.

Reply:
Done